TrieUnitTest/UnitTest1.cs

```diff
@@ -0,0 +1,83 @@
+using homework2;
+using NUnit.Framework;
+ 
+namespace TrieTest;
+ 
+public class Tests
+{
+    private Trie trie;
+    [SetUp]
+    public void Setup()
+    {
+        trie = new Trie();
+    }
+ 
+    [Test]
+    public void AddingStoredStringShallDoNothing()
+    {
+        trie.Add("string");
+        trie.Add("string");
+        if (trie.Size == 1 && trie.Contains("string"))
+        {
+            Assert.Pass();
+        }
+        Assert.Fail();
```

Comment:
```suggestion
        Assert.AreEqual(1, trie.Size);
        Assert.IsTrue(trie.Contains("string"));
```
Вообще Assert.Pass и Assert.Fail используются очень редко, используются методы сравнения.